run_ci_tests.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```

Comment:
I don't follow why this, the `Dockerfile`, and the `run_tests.sh` script is needed? We already get the server through
https://github.com/jonhoo/openssh-rs/blob/3f80def1d80451014d2718d9b855ca907ced039c/azure-pipelines.yml#L145

Is it just so that the tests can be run locally? I'd then rather just give people the in-theory single command they'd need to run to run the appropriate server — everything in this script shouldn't be needed I don't think?

Reply:
That script along with run_ci_test.sh is used to run the integration test locally so that I can discover errors earlier.